### PR TITLE
Add Footer to Documentation

### LINKED ISSUE #183: Add footer to documentation page
Documentation needs Alteryx footer at the bottom of the page. 

I have the SVG for "Alteryx Open Source", please reach out and I can share it! 

- Banner color = HEX #0D2345
- Github / Twitter icons moved to footer 
- Copyright changed and added to footer (shown in image). Current documentation says `© Copyright 2020, Feature Labs, Inc..
`
- Contrary to artwork shown, the footer should only show up when you scroll to the very bottom of the page

image

### PR DESCRIPTION
**Pull request by jeff-hernandez**
Closes #183 by adding a footer to [the documentation](https://compose.alteryx.com/en/update_footer/).

### CODE REVIEW

**Comment by tyler3991** [comment]
Jeff, looks great! One small thing: The SVGs look a little funky. I'm going to redo those SVGs and send them to you. Can you swap them out once I do that?

**Comment by jeff-hernandez** [comment]
@tyler3991 yeah sounds good

File: docs/source/_templates/layout.html [hunk 1]
```diff
@@ -3,10 +3,39 @@
...
+    </div>
+    <div class="footer-cell-2">
+      <a href="https://github.com/FeatureLabs/compose#readme" target="_blank">
+        <img class="footer-image-github" src="_static/images/github.svg" alt="GitHub">
+      </a>
+      <a href="https://twitter.com/alteryx?lang=en" target="_blank">
+        <img class="footer-image-twitter" src="_static/images/twitter.svg" alt="Twitter">
+      </a>
+    </div>
+    <div class="footer-cell-3">
```
Comment (tyler3991) [inline docs/source/_templates/layout.html:32]: Is there any way to align the "Alteryx Open Source", Copyright, and the line? The footer line is a little bit further left

Comment (jeff-hernandez) [reply docs/source/_templates/layout.html:32]: yeah, the footer line has been updated

**Review by tyler3991 — APPROVED** [review]
nice!

**Pull request merged**